cli/src/main/java/keywhiz/cli/configs/AddOrUpdateActionConfig.java

```diff
@@ -80,6 +83,10 @@ public long getExpiry() {
     return 0;
   }
 
+  public String getOwner() {
+    return owner;
+  }
```

Comment:
There are several null checks on the Java side that determine whether we should act on the owner or not, so, assuming that the empty string is distinct from null as far as serialization is concerned, I think it would actually break things.